### ISSUE TITLE
Generating browser application bundles (phase: setup)...Processing legacy "View Engine" libraries:

### ISSUE THREAD

**Issue by Yamesh1994** [swimlane/ngx-datatable#2067]
ng serve 
Generating browser application bundles (phase: setup)...Processing legacy "View Engine" libraries:
- @swimlane/ngx-datatable [es2015/esm2015] (git+https://github.com/swimlane/ngx-datatable.git)

Encourage the library authors to publish an Ivy distribution.

Please update Package

**Comment by micahstubbs** [comment]
also facing this issue

**Comment by micahstubbs** [comment]
I will attempt to update ngx-datatable to Angular 13 to resolve this issue.

**Comment by micahstubbs** [comment]
It looks like the issue is fixed in this commit: https://github.com/swimlane/ngx-datatable/commit/d71246ad6cc3d61a87049712c761ceb3ee07b3c8

Use this line in your package.json to fix:

```json
"@swimlane/ngx-datatable": "swimlane/ngx-datatable#d71246ad6cc3d61a87049712c761ceb3ee07b3c8",
```

**Comment by SwissKerim** [comment]
> It looks like the issue is fixed in this commit: [d71246a](https://github.com/swimlane/ngx-datatable/commit/d71246ad6cc3d61a87049712c761ceb3ee07b3c8)
> 
> Use this line in your package.json to fix:
> 
> ```json
> "@swimlane/ngx-datatable": "swimlane/ngx-datatable#d71246ad6cc3d61a87049712c761ceb3ee07b3c8",
> ```

does this work for you?

I got a "git dep preparation failed"

**Comment by micahstubbs** [comment]
@SwissKerim, yes. That's strange.  These steps might help resolve the `git dep...` error you see: https://stackoverflow.com/a/66460379/1732222
